ansible/roles/invoker/tasks/deploy.yml

```diff
@@ -129,6 +129,12 @@
   set_fact:
     invoker_args: "{{ invoker.arguments }} {{ invoker.jmxremote.jvmArgs }}"
 
+- name: stop and remove the invoke container if it's currently running
+  docker_container:
```

Comment:
these changes are orthogonal to the pr no?

Reply:
Oops, yes.  They snuck in because I got tired of manually stopping and removing the container when running multiple tests.  I'll pull them from this branch and file a separate PR.